Doc/tutorial/controlflow.rst

```diff
@@ -305,7 +305,7 @@ The simplest form compares a subject value against one or more literals::
                 return "Something's wrong with the internet"
 
 Note the last block: the "variable name" ``_`` acts as a *wildcard* and
-never fails to match. If no case matches, none of the branches is executed.
+never fails to match.
```

Comment:
> _ branch is executed when no other branch matches

This is assuming that there is a wildcard branch. For instance:

```py
>>> x = 123
>>> match x:
...     case 2: print(3)
...     case 3: print(4)
...
<nothing>
```

For the specific example, it's indeed confusing but generally speaking, it's correct. I suggest keeping the mention but put it after the "Only the first pattern that matches
gets executed and it can also extract components (sequence elements
or object attributes) from the value into variables."

Reply:
me did it